components/Image/ui-world-map.tsx

```diff
@@ -228,39 +238,90 @@ export default function UIWorldMap({ section }: { section: uiTypes.UISectionObj
+    console.log(`Image scale for moveleft is: ${imgScale}`)
+    /*     let xDiff = -50/imgScale
+    console.log(`xdiff is : ${xDiff}`)
+    if (imgLeft + xDiff <= leftLimit) {
+      setImgLeft(leftLimit)
+    } else if (imgLeft + xDiff >= 0) {
+      setImgLeft(0)
+    } else setImgLeft(imgLeft + xDiff) */
+    setImgLeft(-2084)
+  }
+
+  const moveTop = () => {
+    let yDiff = -50 * imgScale
+    if (imgTop + yDiff <= topLimit) {
+      setImgTop(topLimit)
+    } else if (imgTop + yDiff >= 0) {
+      setImgTop(0)
+    } else setImgTop(imgTop + yDiff)
+  }
 
   return (
     <div>
-      <div className="portrait">
+      <div style={{ overflow: 'hidden', height: '350px', width: '350px', touchAction: 'none' }}>
         <div
           ref={divRef}
-          className="wrapper"
-          onMouseUp={handleMouseUp}
-          onMouseMove={handleMouseMove}
-          onTouchEnd={handleMouseUp}
-          onMouseDown={handleMouseDown}
-          onTouchStart={handleMouseDown}
-          onTouchMove={handleMouseMove}
-          onMouseLeave={handleMouseLeave}
-          onDoubleClick={handleDoubleClick}
+          style={{ overflow: 'hidden', height: '350px', width: '350px', touchAction: 'none' }}
         >
           <img
+            onMouseUp={handleMouseUp}
+            onMouseMove={handleMouseMove}
+            onTouchEnd={handleMouseUp}
+            onMouseDown={handleMouseDown}
+            onTouchStart={handleMouseDown}
+            onTouchMove={handleMouseMove}
+            onMouseLeave={handleMouseLeave}
+            onDoubleClick={handleDoubleClick}
             ref={imgRef}
             src={`data:image/jpeg;base64,${images[0]}`}
             style={{
               transform: `translate(${imgLeft}px, ${imgTop}px)`,
-              height: `${scHeight}px`,
-              width: `${scWidth}px)`,
-              transformOrigin: `top left`,
+              transformOrigin: 'top left',
+              width:`2434px`,
+              height:`1544px`
             }}
             onLoad={handleImageLoad}
           />
         </div>
       </div>
-      <span>{`imgLeft: ${imgLeft}px `}</span>
-      <span>{`imgTop: ${imgTop}px  `}</span>
+      <span>
+        <b>{`scale: ${imgScale}  `}</b>
+      </span>
+      <span>
+        <b>{`Dist: ${touchDist}  `}</b>
+      </span>
+      <span>
+        <b>{`curH: ${scHeight}  `}</b>
+      </span>
+      <span>
+        <b>{`curW: ${scWidth}  `}</b>
+      </span>
+      <span>
+        <b>{`imgL: ${imgLeft}  `}</b>
+      </span>
+      <span>
+        <b>{`imgT: ${imgTop}  `}</b>
+      </span>
+      <span>
+        <b>{`limL: ${leftLimit}  `}</b>
+      </span>
+      <span>
+        <b>{`limT: ${topLimit}  `}</b>
+      </span>
+      <button
+        style={{ width: 40, height: 20, backgroundColor: `blue` }}
+        onClick={moveLeft}
+      ></button>
+      <button
+        style={{ width: 40, height: 20, backgroundColor: `orange` }}
+        onClick={moveTop}
+      ></button>
```

Comment:
Let's make these styled components.

- [ ] Make a Kanban task to setup styled components in ui-world-map.tsx